iputil/geo/geo.go

```diff
@@ -101,12 +106,29 @@ func (g *geoip) City(ip net.IP) (City, error) {
 	if c, exists := record.City.Names["en"]; exists {
 		city.Name = c
 	}
+	if len(record.Subdivisions)>0 {
+		if c, exists := record.Subdivisions[0].Names["en"]; exists {
+			city.RegionName = c
+		}
+		if record.Subdivisions[0].IsoCode != "" {
+			city.RegionCode = record.Subdivisions[0].IsoCode
+		}
+	} 
 	if !math.IsNaN(record.Location.Latitude) {
 		city.Latitude = record.Location.Latitude
 	}
 	if !math.IsNaN(record.Location.Longitude) {
 		city.Longitude = record.Location.Longitude
 	}
+	if record.Location.TimeZone != "" {
+		city.Timezone = record.Location.TimeZone
+	}
+	if record.Location.MetroCode > 0 && record.Country.IsoCode == "US" {
```

Comment:
Why the `record.Country.IsoCode == "US"` check?

Reply:
The metrocode is US only according to maxmind doc. See https://maxmind.github.io/GeoIP2-dotnet/doc/v2.7.1/html/P_MaxMind_GeoIP2_Model_Location_MetroCode.htm

I think we should add a comment about this and maybe the link.